src/client/mesh_generator_thread.cpp

```diff
@@ -103,11 +117,9 @@ bool MeshUpdateQueue::addBlock(Map *map, v3s16 p, bool ack_block_to_server, bool
 		neighbors but get them if they aren't already cached)
 	*/
```

Comment:
this comment isn't even accurate anymore, maybe you can use the occasion to fix it